src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/StartupScripts/AzureRmRecoveryServicesSiteRecoveryStartup.ps1

```diff
@@ -0,0 +1,95 @@
+    "Remove-ASRServicesProvider" = "Remove-AzureRmRecoveryServicesAsrServicesProvider";
+    "Remove-ASRStorageClassificationMapping" = "Remove-AzureRmRecoveryServicesAsrStorageClassificationMapping";
+    "Remove-ASRvCenter" = "Remove-AzureRmRecoveryServicesAsrvCenter";
+    "Restart-ASRJob" = "Restart-AzureRmRecoveryServicesAsrJob";
+    "Resume-ASRJob" = "Resume-AzureRmRecoveryServicesAsrJob";
+    "Set-ASRAlertSetting" = "Set-AzureRmRecoveryServicesAsrAlertSetting";
+    "Set-ASRNotificationSetting" = "Set-AzureRmRecoveryServicesAsrAlertSetting";
+    "Set-ASRReplicationProtectedItem" = "Set-AzureRmRecoveryServicesAsrReplicationProtectedItem";
+    "Set-ASRVaultContext" = "Set-AzureRmRecoveryServicesAsrVaultContext";
+    "Set-ASRVaultSettings" = "Set-AzureRmRecoveryServicesAsrVaultContext";
+    "Set-AzureRmRecoveryServicesAsrNotificationSetting" = "Set-AzureRmRecoveryServicesAsrAlertSetting";
+    "Set-AzureRmRecoveryServicesAsrVaultSettings" = "Set-AzureRmRecoveryServicesAsrVaultContext";
+    "Start-ASRApplyRecoveryPoint" = "Start-AzureRmRecoveryServicesAsrApplyRecoveryPoint";
+    "Start-ASRCommitFailover" = "Start-AzureRmRecoveryServicesAsrCommitFailoverJob";
+    "Start-ASRCommitFailoverJob" = "Start-AzureRmRecoveryServicesAsrCommitFailoverJob";
+    "Start-ASRFO" = "Start-AzureRmRecoveryServicesAsrUnplannedFailoverJob";
+    "Start-ASRPFO" = "Start-AzureRmRecoveryServicesAsrPlannedFailoverJob";
+    "Start-ASRPlannedFailoverJob" = "Start-AzureRmRecoveryServicesAsrPlannedFailoverJob";
+    "Start-ASRResyncJob" = "Start-AzureRmRecoveryServicesAsrResynchronizeReplicationJob";
+    "Start-ASRResynchronizeReplicationJob" = "Start-AzureRmRecoveryServicesAsrResynchronizeReplicationJob";
+    "Start-ASRSwitchProcessServerJob" = "Start-AzureRmRecoveryServicesAsrSwitchProcessServerJob";
+    "Start-ASRTestFailoverCleanupJob" = "Start-AzureRmRecoveryServicesAsrTestFailoverCleanupJob";
+    "Start-ASRTestFailoverJob" = "Start-AzureRmRecoveryServicesAsrTestFailoverJob";
+    "Start-ASRTFO" = "Start-AzureRmRecoveryServicesAsrTestFailoverJob";
+    "Start-ASRTFOCleanupJob" = "Start-AzureRmRecoveryServicesAsrTestFailoverCleanupJob";
+    "Start-ASRUnplannedFailoverJob" = "Start-AzureRmRecoveryServicesAsrUnplannedFailoverJob";
+    "Stop-ASRJob" = "Stop-AzureRmRecoveryServicesAsrJob";
+    "Update-ASRPolicy" = "Update-AzureRmRecoveryServicesAsrPolicy";
+    "Update-ASRProtectionDirection" = "Update-AzureRmRecoveryServicesAsrProtectionDirection";
+    "Update-ASRRecoveryPlan" = "Update-AzureRmRecoveryServicesAsrRecoveryPlan";
+    "Update-ASRServicesProvider" = "Update-AzureRmRecoveryServicesAsrServicesProvider";
+    "Update-ASRvCenter" = "Update-AzureRmRecoveryServicesAsrvCenter";
+}.GetEnumerator() | Select @{Name='Name'; Expression={$_.Key}}, @{Name='Value'; Expression={$_.Value}} | `
+ForEach-Object {if (!(Get-Alias -Name $_.Name -ErrorAction Ignore)) {Set-Alias -Name $_.Name -Value $_.Value -Description "AzureAlias" -Scope Global}} | Out-Null
```

Comment:
These aliases won't work in NetStandard (Az). No need to change them now, but I'm just commenting on it so you're aware.

Reply:
For Az how can we make these alias to work ?